README.md

```diff
@@ -27,7 +32,7 @@ $.ajax({
 ```
 ### Question 5
 In an AJAX POST request, what does the argument of the `.done()` callback - in the example below, that would be `data` - represent? (Hint: it's not the same as in Question 4.)
-
+the data inputted from the post request, =>  data: { user: { name: "Anna", about: "instructor", email: "hi@gmail.com" } }
```

Comment:
Close. In a POST request, the "data" argument is actually the response from the server. By convention, though, the response is the objected that was just POSTed.